packages/babel-plugin-proposal-export-default-from/src/index.ts

```diff
@@ -24,19 +24,13 @@ export default declare(api => {
           return;
         }
 
-        const nodes = [
+        path.insertBefore(
           t.exportNamedDeclaration(
             null,
             [t.exportSpecifier(t.identifier("default"), exported)],
             t.cloneNode(source),
           ),
-        ];
-
-        if (specifiers.length >= 1) {
```

Comment:
This condition was always `true` because if the length is 0 then the `.every` call above returns true.